buildSrc/build.gradle

```diff
@@ -24,4 +24,6 @@ dependencies {
     compile "no.uio.ifi:crypt4gh:1.2.0"
     compileOnly 'org.projectlombok:lombok:1.18.6'
     annotationProcessor 'org.projectlombok:lombok:1.18.6'
+    compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
```

Comment:
Can you make it the same ways as other dependencies are declared? Or re-declare other dependencies in a new way. I'd just prefer to have it in the same style.